lib/client/chart.js

```diff
@@ -235,15 +280,16 @@ function init (client, d3, $) {
     var dataRange = client.dataExtent();
     var chartContainerRect = chartContainer[0].getBoundingClientRect();
     var chartWidth = chartContainerRect.width;
-    var chartHeight = chartContainerRect.height - padding.bottom;
+    var chartHeight = chartContainerRect.height - PADDING_BOTTOM;
 
     // get the height of each chart based on its container size ratio
     var focusHeight = chart.focusHeight = chartHeight * .7;
-    var contextHeight = chart.contextHeight = chartHeight * .2;
+    var contextHeight = chart.contextHeight = chartHeight * .3;
```

Comment:
Re the `open-` drawing height bugs: Was there a particular reason this was changed from `.2` to `.3`? Wondering if perhaps how it adds up here, plus padding, could be overflowing the height box, or something along those lines.